README.md

```diff
@@ -44,6 +44,7 @@ Have fun creating art!
 | ![People Blending](results/gates.jpg) | [@shwars][shwars] | [CognitivePortrait.ipynb](CognitivePortrait.ipynb) | A number of photographs are aligned by their eyes and blended together to create an "average" photograph of a person |
 | ![2Face](results/gates_2face.jpg) | [@shwars][shwars] | [CognitivePortrait-2face.ipynb](CognitivePortrait-2face.ipynb) | Photographs are grouped into two sets of different ages, and blended side-by-side |
 | ![FaceCircle](results/gates_circ.jpg) | [@shwars][shwars] | [CognitivePortrait-Circle.ipynb](CognitivePortrait-Circle.ipynb) | Aligning faces to that eyes lie on a circle, creating visual whirlpool |
+| ![MASONS EVERYWHERE](results/gates_is_mason.jpg) | [@semenovdv][semenovdv] | [CognitivePortrait-Masons.ipynb](CognitivePortrait-Masons.ipynb) | Aligning the pupils into a triangle, thereby proving that he is a Mason |
```

Comment:
Please, remove mentioning **masons** here, PR again, and let's get it merged to the original!